problem_builder/public/css/dashboard.css

```diff
@@ -24,7 +32,7 @@
     min-width: 4em;
     text-align: right;
     padding-right: 5px;
-    border-right: 0.6em solid transparent;
+    border-right: 2em solid transparent;
```

Comment:
You might want to just consider defining some baseline variables then having sizes be relative to that (eg baseline/4, baseline/10 or whatever). That way you don't have to do fussy math all the time and worry that numbers like 1.875 are correct.

Perhaps a future improvement to think about.